spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -103,6 +104,10 @@ object SuspiciousConnectsArgumentParser {
       action((x, c) => c.copy(maxResults = x)).
       text("number of most suspicious connections to return")
 
+    opt[String]("ldaImplementation").required().valueName("< LDAC | sparkLDA >").
```

Comment:
Are we making a custom call? My mindset was that the switch was for extended testing.